spec/delay/sidekiq_spec.rb

```diff
@@ -35,8 +35,8 @@ class Sidekiq
 
     context "with default options" do
       it "enqueues to default queue" do
-        options = Rollbar::Delay::Sidekiq::OPTIONS.merge('args' => payload)
-        ::Sidekiq::Client.should_receive(:push).with options
+        options = Rollbar::Delay::Sidekiq::OPTIONS.merge('args' => [payload])
+        ::Sidekiq::Client.should_receive(:push).with(options){ true }
```

Comment:
Since we are recently now using the return value of `::Sidekiq::Client.push`, this needs to mock a return value.